index.html

```diff
@@ -2595,6 +2595,137 @@ <h3>Data Schemas</h3>
+          </a> registry.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY contain
+          additional properties.
+        </p>
+        <p>
+          In any object in the <code>credential.credentialSubject</code>
+          that contains a [[URL]], a property named
+          <code>integrity</code> may be included with the integrity
+          information as specified above, with the [[URL]] in the
+          <code>id</code> property. 
+        </p>
+        <p>
+          Any objects for which selective disclosure is desired SHOULD NOT
+          be included as an object in the <code>resourceIntegrity</code>
+          array.
+        </p>
+        <p>
+          Implementers SHOULD consult appropriate sources, such as the 
+          <a href="https://www.iana.org/assignments/named-information/named-information.xhtml">IANA
+          Named Information Hash Algorithm Registry</a> to ensure that they
+          are chosing a current and reliable hash algorithm.  At the time of
+          this writing `sha-256` SHOULD be considered the minimum strength
+          hash algorithm for use by implemnters.
+        </p>
+        <p class="issue">
+          The working group is discussing if we will adopt subresource
+          integrity as defined in [[SRI]] is adopted into the [[JSON-LD]]
+          specification as noted in that specifications <a
+          href="https://www.w3.org/TR/json-ld11/#security">current
+          security considerations</a> of that specification, this hash in
+          the VC can serve as an additional check towards ensuring that a
+          cached context used when issuing the VC matches the remote
+          resource.
+        </p>
+        <p>
+          <aside
+            class="example"
+            title="resource integrity"
+          >
+            <p>An example of a resource integrity object referencing contexts</p>
+            <pre>
+              "resourceIntegrity": [{
+                "id": "https://www.w3.org/ns/credentials/v2",
+                "integrity": "sha384-lHKDHh0msc6pRx8PhDOMkNtSI8bOfsp4giNbUrw71nXXLf13nTqNJoRp3Nx+ArVK", 
+              },{  
+                "id": "https://www.w3.org/ns/credentials/examples/v2",  
+                "integrity": "sha384-zNNbQTWCSUSi0bbz7dbua+RcENv7C6FvlmYJ1Y+I727HsPOHdzwELMYO9Mz68M26", 
+              }]
+            </pre>
+          </aside>
+        </p>
+        <p>
+          <aside
+            class="example"
+            title="resource integrity over image"
+          >
+            <p>An example of a resource integrity object in a credentialSubject refering to an image</p>
+            <pre>
+              "credentialSubject": {
+                "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+                "image": {
+                    "id": "https://university.example/images/58473",
+                    "integrity": "sha384-ZfAwuJmMgoX3s86L7x9XSPi3AEbiz6S/5SyGHJPCxWHs5NEth/c5S9QoS1zZft+J"
```

Comment:
Would be nice to have a real world working example, that also includes the optional fields. I suggest 
```
"image": {
  "id": "https://www.w3.org/StyleSheets/TR/2016/logos/W3C",
  "integrity": "sha384-+EzHAeRZGsx0HtzBd6u0Qy5ezQPL1yUD75E3TZjI/r9UJmecM25khdGE+VWaYJUW",
  "mediaType": "application/svg+xml",
  "timestamp": "2023-06-16T17:10:24Z"
}
```

Reply:
[4b8ffbf](https://github.com/w3c/vc-data-model/pull/1140/commits/4b8ffbf7b4623c9be5474601470f9d358c75071a) adds this, but it may trigger some folks

Reply:
Bikeshedding: `timestamp` doesn't quite fit... Imagine it as a single property associated with the subject:

```
<https://www.w3.org/StyleSheets/TR/2016/logos/W3C> timestamp "2023-06-16T17:10:24Z" .
```

That resource has a timestamp, but it doesn't say what that means... was it "the last time the resource was accessed?" is it "the last time the resource was modified" or maybe it's "when the resource was created". We should differentiate... `accessedTimestamp`, `lastModifiedTimestamp`, `createTimestamp` -- we should be more specific with the name.

Don't know how useful it is, either. You have a signature on the payload (presumably), so you can probably just infer the timestamp via that. When would that not be a safe thing to do?